app/forms/search_form.rb

```diff
@@ -4,6 +4,11 @@ class SearchForm
   include ActiveRecord::AttributeAssignment
 
   Option = Struct.new(:value, :label)
+  ApplicationType = Struct.new(:id, :name)
+  APPLICATION_TYPES = [
+    ApplicationType.new('crm4', 'Prior authority'),
```

Comment:
These user-facing strings should be in locale files, not hard-coded into the model